web-client/README.md

```diff
@@ -50,24 +67,13 @@ We follow the flat File-System Architecture, route definitions currently live in
 
 #### Data
 
-Instrumentation data comes in the app through the Web Socket connection and is stored in a `ws/store.ts` before be made available via the `DataProvider` context.
-
-So, there are 2 contexts that are important:
-
-- `WSContext`: provides the WebSocket instance and the current connection state.
-- `DataProvider`: provides the data store to used.
-
-Both of them are made available to every child component via their custom hooks:
-
-- `useSocketData`: returns the data store.
-- `useWs`: returns the WebSocket and its connection state.
-
-#### Web Sockets
-
-Communication with the Rust instrumentation is done via Web Sockets and uses the SolidJS Web Sockets primitive, with SolidJS EventListeners primitive to convert DOM Events into signals.
+Instrumentation data comes in the app through 4 different stream clients.
```

Comment:
```suggestion
Instrumentation data comes in the app through 5 different stream clients.
```

Reply:
Or is one of them not a stream client. Either way it is weird to have the number 4 and then list  5 items